templates/conf.d/nginx.conf.erb

```diff
@@ -74,7 +74,11 @@ http {
   default_type  application/octet-stream;
 <% if @log_format -%>
 <% @log_format.sort_by{|k,v| k}.each do |key,value| -%>
-  log_format <%= key %> '<%= value %>';
+  <%- if value.is_a?(String) -%>
+  log_format <%= key %> <%= value.inspect %>;
+  <%- else -%>
+  log_format <%= key %> <%= "escape=#{value['escape']} " if value['escape'] %><%= value['format'].inspect %>;
```

Comment:
Should this be swapped around? Feels like `is_a?(Hash)` would be more robust, though it probably doesn't matter.

Reply:
No objection.  Just updated the PR accordingly.